cmd/cluster-agent/subcommands/compliance/command.go

```diff
@@ -6,34 +6,30 @@
 //go:build !windows && kubeapiserver
 // +build !windows,kubeapiserver
 
-package app
+// Package compliance implements 'cluster-agent compliance'.
+package version
 
 import (
-	"github.com/spf13/cobra"
-
+	"github.com/DataDog/datadog-agent/cmd/cluster-agent/command"
 	"github.com/DataDog/datadog-agent/cmd/security-agent/app/subcommands/check"
 	"github.com/DataDog/datadog-agent/comp/core"
 	"github.com/DataDog/datadog-agent/comp/core/config"
 	"github.com/DataDog/datadog-agent/comp/core/log"
+
+	"github.com/spf13/cobra"
 )
 
-var (
-	complianceCmd = &cobra.Command{
+// Commands returns a slice of subcommands for the 'cluster-agent' command.
+func Commands(globalParams *command.GlobalParams) []*cobra.Command {
+	complianceCmd := &cobra.Command{
 		Use:   "compliance",
-		Short: "Compliance utility commands",
+		Short: "compliance utility commands",
 	}
-)
 
-func init() {
-	bundleParams := core.BundleParams{
-		ConfigParams: config.NewParams(
-			"",
-			config.WithConfFilePath(confPath),
-			config.WithConfigName("datadog-cluster"),
-		),
-		LogParams: log.LogForOneShot(string(loggerName), "off", true),
-	}
+	complianceCmd.AddCommand(check.Commands(core.BundleParams{
+		ConfigParams: config.NewClusterAgentParams(""),
+		LogParams:    log.LogForOneShot("CLUSTER", "off", true),
```

Comment:
can we reuse the `const logerName = "CLUSTER"` defined here: https://github.com/DataDog/datadog-agent/pull/15009/files#diff-743e6399c0f80ca0502958e39dbfd259e8df0889ff5ae1a04717dfed3b76e09aR19